less/index.less

```diff
@@ -48,12 +48,119 @@ table {
 	border-spacing: 0;
 }
 
+// Variables
+@font-stack: Arial, "sans-serif";
+@my-blue: rgb(176, 237, 237);
+
 /* Set every element's box-sizing to border-box */
 * {
     box-sizing: border-box;
 }
 
+nav {
+	border: 2px solid black;
+	margin: 20px 0;
+	padding: 20px 0;
+	display: flex;
+	justify-content: space-evenly;
+	font-size: 2rem;
+	background: white;
+}
+
+html {
+	box-sizing: border-box;
+	font-size: 62.5%
+  }
+
 html, body {
-    height: 100%;
+	height: 100%;
+	box-sizing: border-box;
+	font-size: 62.5%;
+	font-family: @font-stack;
 }
 
+body {
+	background:@my-blue;
+}
+
+h1, h2, h3, h4, h5 {
+	font-size: 2rem;
+	padding: 1rem;
+	text-align: center;
+  }
+  h1 {
+	  font-size: 3rem;
+	  text-align: center;
+  }
+  p {
+	  font-size: 2rem;
+  }
+
+  .container {
+	  max-width: 800px;
+	  margin: 0 auto;
+	//   font-size: 1.6rem;
+	  display:flex;
+  }
+
+  .image1 {
+	max-width: 800px;
+	width: 100%;
+	// margin-left: 100pt;
+	
+  img {
+		// width: 100%;
+		// justify-content: center;
+		// max-width: 800px;
+		// width: 100%;
+		// height: auto;
+		// position: center;
+		// align-content: center;
+		// align-items: center;
+		// // padding: 0 50px;
+		// // margin: 0 200px;
+		// display: flex;
+
+		
+		display: block;
+		margin-left: auto;
+		margin-right: auto;
+		width: 100%;
+		height: auto;
+  	}
+  }
+  .main {
+	border-bottom: 2px solid black;
+	margin: 20px 0;
+	padding: 20px ;
+	background: white;
+	max-width: 800px;
+	justify-content: center;
+	display: block;
+		margin-left: auto;
+		margin-right: auto;
```

Comment:
instead of these you could simply change margin 20px 0; at the top to margin: 20px auto; for the same effect!